base/linalg/triangular.jl

```diff
@@ -1075,6 +1078,7 @@ function logm{T<:Union{Float64,Complex{Float64}}}(A0::UpperTriangular{T})
     return UpperTriangular(Y)
 
 end
+logm(A::LowerTriangular) = logm(A.').'
```

Comment:
I think it is, I just use the fact that if you transpose an `UpperTriangular` you get a `LowerTriangular`.